### ISSUE TITLE
Score to track github issues for component changes

### ISSUE THREAD

**Issue by joshklm** [OAGi/Score#1533]
When user moves a component updates to Candidate state, Score tool should popup an input dialog for user to add the Github issue(s) and comments. The entry will be saved for future review/assessment, and also for a notification when the user tries to revert the component back from the Candidate state. This will solve the problem of potentially losing changes to components tied to multiple Github issues.

**Comment by joshklm** [comment]
This issue is generated during the 8/11 OAGi Architecture Meeting with @joshklm @michaelfigura @jim-wilson-kt @sfohn regarding lost changes related to several Github issues that were previously approved for OAGIS 10.9.x releases.

**Comment by kbserm** [comment]
@joshklm could you provide more detail about the notification.

**Comment by joshklm** [comment]
@kbserm Yes. @dubnemo just nailed it with his inflation adjusted opinion. :) I'm not sure if it's called Cancel button. I think it's called 'Revert' to get to WIP State from Candidate state. During the meeting on 8/11 we found out that the person who is reverting the component from Candidate state back to WIP may not realize (or simply forgot) that the component had updates to cover multiple github issues for the delivery under the target OAGIS release.
